content/applications/finance/accounting/taxes/avatax/avatax_use.rst

```diff
@@ -11,15 +11,15 @@ Tax calculation
 
 Automatically calculate taxes on Odoo quotations and invoices with AvaTax by confirming the
 documents during the sales flow. Alternatively, calculate the taxes manually by clicking the
-:guilabel:`Compute taxes using Avatax` button while these documents are in draft mode.
+:guilabel:`Compute Taxes` button while these documents are in draft mode.
```

Comment:
```suggestion
:guilabel:`Compute Taxes` button, while these documents are in draft mode.
```

Reply:
do you think 'draft mode' or 'draft stage' is more accurate? Totally your call...